ui/components/multichain/connected-site-popover/connected-site-popover.tsx

```diff
@@ -30,10 +31,13 @@ type ConnectedSitePopoverProps = {
   referenceElement?: RefObject<HTMLElement>;
 };
 
-export const ConnectedSitePopover = forwardRef<
-  HTMLDivElement,
-  ConnectedSitePopoverProps
->(({ isOpen, isConnected, onClick, onClose, referenceElement }) => {
```

Comment:
I don't think we need to forward refs for this component unless we need to get the position or dimensions of the Popover. We can use the `referenceElement` prop to pass the ref of the component that triggers the Popover. Removing for simplicity.